dockerfiles/Dockerfile.gunicorn

```diff
@@ -1,8 +1,10 @@
-# Dockerfile for running titiler application with gunicorn server
-# Size ~1.4GB
+# Dockerfile for running titiler application with uvicorn server
 ARG PYTHON_VERSION=3.9
 
-FROM tiangolo/uvicorn-gunicorn:python${PYTHON_VERSION}
+FROM bitnami/python:${PYTHON_VERSION}
+RUN apt update && apt upgrade -y \
+  && apt install curl -y \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
@FischerLGLN Tiangolo image has more than just gunicorn 

I guess the easiest way might just to update https://github.com/vincentsarago/containers/tree/main/uvicorn-gunicorn with the bitnami docker images.

Reply:
You're fast. 

Do you mean this file?
https://github.com/vincentsarago/containers/blob/main/uvicorn-gunicorn/python3.9.dockerfile

I think its mainly the start.sh and start-reload.sh files as well as the guvicorn config, but they aren't needed in a kubernetes platform.
Idea:
We just say guvicorn is for dev and normal use and the uvicorn image is for kubernetes.
I would handle in this PR just the Uvicorn Part and change in kubernetes deployment.

Reply:
> I would handle in this PR just the Uvicorn Part and change in kubernetes deployment.

sounds good to me

Reply:
@vincentsarago Gunicorn is semantically untouched and with this PR I get my updated uvicorn :)

Reply:
🙏

Reply:
FYI: https://github.com/vincentsarago/containers/pull/3